react/components/admin/institutional/Form/withContentContext.tsx

```diff
@@ -103,9 +106,44 @@ function withContentContext<T>(
                   return <Loader />
                 }
 
-                const blockId = dataTemplates!.availableTemplates.filter(
+                const elegibleTemplates = dataTemplates!.availableTemplates.filter(
                   (template: Template) => template.id !== interfaceId
-                )[0].id
+                )
+
+                if (!elegibleTemplates.length) {
+                  return (
+                    <div>
+                      <EmptyState
+                        title={
+                          <FormattedMessage
+                            id="admin/pages.admin.content.not-supported.title"
+                            defaultMessage="This feature is not available :/"
+                          />
+                        }
+                      >
+                        <p>
+                          <FormattedMessage id="admin/pages.admin.content.not-supported.description" />
+                        </p>
```

Comment:
⛏ I personally prefer using render props in cases like that one, because the inner `span` isn't really necessary.
```tsx
<FormattedMessage id="admin/pages.admin.content.not-supported.description">
  {message => <p>{message}</p>}
</FormattedMessage>
```

Reply:
There's no extra `span`: https://github.com/vtex-apps/render-runtime/blob/master/react/components/RenderProvider.tsx#L898